builds/misc/images-release.yaml

```diff
@@ -7,7 +7,7 @@ jobs:
     # The code sign steps will fail unless we explicitly say to use dotnet 2 before.
     # This means we have to toggle back and forth between primary dotnet installations for the sign and build.
     # TODO: Investigate why we have to toggle primary installs on linux, when we didn't have to do this on windows (now removed).
-    timeoutInMinutes: 120
+    timeoutInMinutes: 170
```

Comment:
how long does it takes? need to increase 50 mins?

Reply:
Yes, the 3 build combined take on average 45min.

Reply:
Make it 180 for now.  Please help to create a PBI to optimize our build time, e.g. divide the work in different job agents and send PBI link to @varunpuranik thanks.